tdrs-backend/tdpservice/search_indexes/parsers/schema_defs/tanf.py

```diff
@@ -1,117 +1,104 @@
-            ('STRATUM', 2, 23, 24, "Numeric"),
-            ('ZIP_CODE', 5, 25, 29, "Alphanumeric"),
-            ('FUNDING_STREAM', 1, 30, 30, "Numeric"),
-            ('DISPOSITION', 1, 31, 31, "Numeric"),
-            ('NEW_APPLICANT', 1, 32, 32, "Numeric"),
-            ('NBR_FAMILY_MEMBERS', 2, 33, 34, "Numeric"),
-            ('FAMILY_TYPE', 1, 35, 35, "Numeric"),
-            ('RECEIVES_SUB_HOUSING', 1, 36, 36, "Numeric"),
-            ('RECEIVES_MED_ASSISTANCE', 1, 37, 37, "Numeric"),
-            ('RECEIVES_FOOD_STAMPS', 1, 38, 38, "Numeric"),
-            ('AMT_FOOD_STAMP_ASSISTANCE', 4, 39, 42, "Numeric"),
-            ('RECEIVES_SUB_CC', 1, 43, 43, "Numeric"),
-            ('AMT_SUB_CC', 4, 44, 47, "Numeric"),
-            ('CHILD_SUPPORT_AMT', 4, 48, 51, "Numeric"),
-            ('FAMILY_CASH_RESOURCES', 4, 52, 55, "Numeric"),
-            ('CASH_AMOUNT', 4, 56, 59, "Numeric"),
-            ('NBR_MONTHS', 3, 60, 62, "Numeric"),
-            ('CC_AMOUNT', 4, 63, 66, "Numeric"),
-            ('CHILDREN_COVERED', 2, 67, 68, "Numeric"),
-            ('CC_NBR_MONTHS', 3, 69, 71, "Numeric"),
-            ('TRANSP_AMOUNT', 4, 72, 75, "Numeric"),
-            ('TRANSP_NBR_MONTHS', 3, 76, 78, "Numeric"),
-            ('TRANSITION_SERVICES_AMOUNT', 4, 79, 82, "Numeric"),
-            ('TRANSITION_NBR_MONTHS', 3, 83, 85, "Numeric"),
-            ('OTHER_AMOUNT', 4, 86, 89, "Numeric"),
-            ('OTHER_NBR_MONTHS', 3, 90, 92, "Numeric"),
-            ('SANC_REDUCTION_AMT', 4, 93, 96, "Numeric"),
-            ('WORK_REQ_SANCTION', 1, 97, 97, "Numeric"),
-            ('FAMILY_SANC_ADULT', 1, 98, 98, "Numeric"),
-            ('SANC_TEEN_PARENT', 1, 99, 99, "Numeric"),
-            ('NON_COOPERATION_CSE', 1, 100, 100, "Numeric"),
-            ('FAILURE_TO_COMPLY', 1, 101, 101, "Numeric"),
-            ('OTHER_SANCTION', 1, 102, 102, "Numeric"),
-            ('RECOUPMENT_PRIOR_OVRPMT', 4, 103, 106, "Numeric"),
-            ('OTHER_TOTAL_REDUCTIONS', 4, 107, 110, "Numeric"),
-            ('FAMILY_CAP', 1, 111, 111, "Numeric"),
-            ('REDUCTIONS_ON_RECEIPTS', 1, 112, 112, "Numeric"),
-            ('OTHER_NON_SANCTION', 1, 113, 113, "Numeric"),
-            ('WAIVER_EVAL_CONTROL_GRPS', 1, 114, 114, "Numeric"),
-            ('FAMILY_EXEMPT_TIME_LIMITS', 2, 115, 116, "Numeric"),
-            ('FAMILY_NEW_CHILD', 1, 117, 117, "Numeric"),
-            ('BLANK', 39, 118, 156, "Spaces"),
-        ]
-    )
-    return family_case_schema
+def t1_schema():
+    """Return a list of rows for T1 records."""
+    return [
+        row(None, 'RecordType', 2, 1, 2, "Alphanumeric"),
+        row('4', 'RPT_MONTH_YEAR', 6, 3, 8, "Numeric",
+            cat2_conditions={'custom': [t1_006, t1_007]}),
+        row('6', 'CASE_NUMBER', 11, 9, 19, "Alphanumeric"),
+        row('2', 'COUNTY_FIPS_CODE', 3, 20, 22, "Numeric"),
+        row('5', 'STRATUM', 2, 23, 24, "Numeric",
+            cat2_conditions={'gt': 0, 'lt': 100}),
```

Comment:
looking at https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_fatal_edits_sections_1_and_4.pdf

> ITEM 5 MUST = 00-99

should `STRATUM` be `gte: 0` and `lt: 100`